PIVX-Labs/MyPIVXWallet
# Add unused lang tool checker

## Abstract

Added a tool to detect unused translation keys in the code.
Updated other scripts so that deleted keys from the template will delete it from other languages as well.
Removed unused keys
Resetted MN_START_FAILED and MN_RESTART_FAILED because they had wrong keys

## Testing
- Add unused translation keys to template and check that `check_unused.py` marks it as unused
- Remove it from the template and check that `update_translation` returns all the files to as they were before

If any errors are found, the PR works unexpectedly, or you have viable suggestions to improve the UX or functionality of the PR, let me know!
